### ISSUE TITLE
w is not defined at RoundRect.exec 

### ISSUE THREAD

**Issue by where-tsukada** [CreateJS/EaselJS#976]
* Version used: @createjs/easeljs 2.0.0-beta.2
* OS & Browser version: MAC OS 10.13.6 (Chrome 66.0.3359.117)

When I tried to draw RoundRect like
    shape.graphics.beginFill("#76ccf7").drawRoundRect(0, 0, 60, 30, 20)

it gave the follwing error.

```
client.js:513 ReferenceError: w is not defined
    at RoundRect.exec (easeljs.module.js:2996)
    at Graphics.draw (easeljs.module.js:2472)
    at Shape.draw (easeljs.module.js:3446)
    at Container.draw (easeljs.module.js:1190)
```
The error happened at   easeljs/display/Graphics.js

```
(G.RoundRect = function(x, y, w, h, radiusTL, radiusTR, radiusBR, radiusBL) {
		this.x = x; this.y = y;
		this.w = w; this.h = h;
		this.radiusTL = radiusTL; this.radiusTR = radiusTR;
		this.radiusBR = radiusBR; this.radiusBL = radiusBL;
}).prototype.exec = function(ctx) {
		var max = (w<h?w:h)/2;
		var mTL=0, mTR=0, mBR=0, mBL=0;
```
constructor seems to be ok, but at exec function, w and h are undefined.

drawCircle or drawRect has no problem.

**Comment by lannymcnie** [comment]
Interesting. 1.0 had the same code issue, but doesn't provide an error.

Thanks for finding, we can post a fix.

**Comment by lannymcnie** [comment]
I posted a fix to the 1.0 source, but am not going to make a minified version, since nothing breaks. Version 2.0 fix coming shortly.

Feel free to post pull-requests for easy fixes like this :)

**Comment by lannymcnie** [comment]
The 2.0 branch has been updated!
